src/client/app/strings/spanish.ts

```diff
@@ -0,0 +1,258 @@
+import { ClientStrings } from './index';
+
+// Whenever there is no localized translation, use the English version.
+// Whenever these two lines are not commented out, it means that there are
+// still strings that need to be localized.
+// Just search for "englishSubstitutesForNotYetTranslated" in this file
+import { getEnglishStrings } from './english';
+const englishSubstitutesForNotYetTranslated = getEnglishStrings();
+
+export function getSpanishStrings(): ClientStrings {
+    const playgroundName = 'Script Lab';
+
+    return {
+        playgroundName: playgroundName,
+        playgroundTagline: 'Codifica ? Ejecuta ? Comparte',
+
+        userId: 'ID de usuario',
+
+        alpha: englishSubstitutesForNotYetTranslated.alpha,
+        beta: englishSubstitutesForNotYetTranslated.beta,
+        production: englishSubstitutesForNotYetTranslated.production,
+
+        run: 'Ejecutar',
+        runInThisPane: 'Ejecuta el código en este panel',
+        runSideBySide: 'Ejecuta el código en un panel adyacente',
+        share: 'Compartir',
+        delete: 'Borrar',
+        close: 'Cerrar',
+        about: 'Acerca de',
+        feedback: 'Retroalimentación',
+        errors: 'Errores',
+        dismiss: 'Ignorar',
+
+        okButtonLabel: 'OK',
+        logoutButtonLabel: 'Cerrar sesión',
+        cancelButtonLabel: 'Cancelar',
+        saveButtonLabel: 'Guardar',
+        moreInfoButtonLabel: 'Más información',
+        importButtonLabel: 'Importar',
+        editorTriggerSuggestContextMenuLabel: 'Sugerir código',
+
+        snippetImportError: 'Error al importar fragmento de código',
+        snippetImportErrorTitle: 'Error en importación',
+        snippetImportErrorBody: 'No pudimos importar el fragmento de código.',
+        reloadPrompt: 'Vuelve a cargar este panel e intenta otro URL o ID.',
+
+        snippetSaveError: 'Error al guardar el fragmento de código',
+        snippetDupeError: 'Error al duplicar el fragmento de código',
+        snippetDeleteError: 'Error al borrar el fragmento de código',
+        snippetDeleteAllError: 'Error al borrar todos los fragmentos de código locales',
+        snippetLoadAllError: 'Error al cargar fragmentos de código locales',
+        snippetRunError: 'Error al ejecutar el fragmento de código',
+        snippetLoadDefaultsError: 'Error al cargar ejemplos',
+
+        snippetNoOfficeTitle: /** NEEDS STRING REVIEW */ 'No se puede ejecutar el fragmento de código',
```

Comment:
@cshashwat , could you please remove all these "needs string review" comments for the Spanish comments (in the sense that they already have been reviewed, by a native speaker) :-)